src/components/profile-card.jsx

```diff
@@ -1,5 +1,42 @@
 import React from 'react';
 
-const ProfileCard = (props) => <div>Profile Card for: {props.username}</div>;
+const url = 'https://mcr-codes-cohorts.herokuapp.com/users/';
+
+class ProfileCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      profile: null,
+    };
+  }
+
+  componentDidMount() {
+    fetch(`${url}${this.props.username}`)
+      .then(response => response.json())
+      .then(data => this.setState({ profile: data.profile }));
+  }
+   
+  
+  render() {
+    console.log(this.state);
+  if (this.state.profile !== null) {
+      return ( 
+    <div>
+      <img src={this.state.profile.avatar} />
```

Comment:
why not use object destructuring to pick these `avatar, login, location, repos` properties off profile.

```
const {avatar, login, location, repos} = this.state.profile;
```